### ISSUE TITLE
btest fails in Vagrant environment

### ISSUE THREAD

**Issue by jeremycline** [fedora-infra/bodhi#2859]
While checking my work on #2858 I found that the tests in the Vagrant environment fail because a non-ascii character is used without a character encoding declared in a number of files:
```
/usr/lib/python2.7/site-packages/py/_path/common.py:377: in visit
    for x in Visitor(fil, rec, ignore, bf, sort).gen(self):
/usr/lib/python2.7/site-packages/py/_path/common.py:429: in gen
    for p in self.gen(subdir):
/usr/lib/python2.7/site-packages/py/_path/common.py:429: in gen
    for p in self.gen(subdir):
/usr/lib/python2.7/site-packages/py/_path/common.py:429: in gen
    for p in self.gen(subdir):
/usr/lib/python2.7/site-packages/py/_path/common.py:419: in gen
    if p.check(dir=1) and (rec is None or rec(p))])
/usr/lib/python2.7/site-packages/_pytest/main.py:511: in _recurse
    ihook = self.gethookproxy(path)
/usr/lib/python2.7/site-packages/_pytest/main.py:413: in gethookproxy
    my_conftestmodules = pm._getconftestmodules(fspath)
/usr/lib/python2.7/site-packages/_pytest/config/__init__.py:386: in _getconftestmodules
    mod = self._importconftest(conftestpath)
/usr/lib/python2.7/site-packages/_pytest/config/__init__.py:417: in _importconftest
    raise ConftestImportFailure(conftestpath, sys.exc_info())
E   ConftestImportFailure: SyntaxError("Non-ASCII character '\\xc2' in file /home/vagrant/bodhi/devel/ci/integration/tests/fixtures/backend.py on line 1, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details", ('/home/vagrant/bodhi/devel/ci/integration/tests/fixtures/backend.py', 1, 0, None))
E     File "/usr/lib/python2.7/site-packages/_pytest/assertion/rewrite.py", line 226, in load_module
E       py.builtin.exec_(co, mod.__dict__)
E     File "/usr/lib/python2.7/site-packages/py/_builtin.py", line 221, in exec_
E       exec2(obj, globals, locals)
E     File "<string>", line 7, in exec2
E     File "/home/vagrant/bodhi/devel/ci/integration/tests/conftest.py", line 21, in <module>
E       from .fixtures.backend import docker_backend, docker_network
```
This is only a problem in Python 2. Was the plan to drop Python 2 support? If so I think the fix here is to just drop that from ``btest``.

**Comment by bowlofeggs** [comment]
Ah I know what the problem is - the new integration test suite itself is Python 3 only, and btest uses the path ```/home/vagrant/bodhi``` to find tests, rather than ```/home/vagrant/bodhi/bodhi/tests```. Easy fix!

**Comment by sebwoj** [comment]
@bowlofeggs Can we close this issue? I have no idea why mergify skip closing it.

**Comment by bowlofeggs** [comment]
@sebwoj Mergify doesn't close issues, GitHub does. I see that the commit message says "Fixes #2859", but the pull request (#2866) doesn't. Perhaps GitHub only closes issues if the pull request description has that text? Not sure.

Either way, it is good to use that text in both the commit message and pull request description so the git log has it and so the pull request is linked to the issue (though it's not hard to get from the commit to the associated pull request).

**Comment by sebwoj** [comment]
> Either way, it is good to use that text in both the commit message and pull request description so the git log has it and so the pull request is linked to the issue (though it's not hard to get from the commit to the associated pull request).

Good to know :)
